### DISCUSSION TITLE
How to set max epochs for an ASR Recipe?

### DISCUSSION THREAD

**Discussion by iamsashank09** [espnet/espnet#4975] [Q&A] [answered]
I'm running an existing ASR recipe on some new data, I'm quite new to this and just trying to understand how I can set the maximum epochs that I want my model to run for?

I am not using Python, I am running it off the shell scripts as shown in the CMU Tutorial of ESPNet. Can someone help me out with what do I add to my stage 11 to set the number of epochs?

**Comment by sw005320** [comment, marked as answer]
An easier way is to edit the config file, e.g., https://github.com/espnet/espnet/blob/master/egs2/an4/asr1/conf/train_asr_transformer.yaml#L4
But you can also pass it to the command line by following 
https://espnet.github.io/espnet/espnet2_tutorial.html#change-the-configuration-for-training
